core/src/main/java/io/aiven/klaw/helpers/db/rdbms/SelectDataJdbc.java

```diff
@@ -1512,6 +1534,16 @@ public Map<String, Map<String, Long>> getSchemaRequestsCounts(
           schemaRequestRepo.findAllSchemaRequestsGroupByStatus(teamId, tenantId);
       updateMap(statusCountsMap, schemaRequestsStatusObj);
     }
+    if (RequestMode.MY_APPROVALS == requestMode) {
+      // Make sure to remove any requests which the requestor can not approve
+      List<Object[]> schemaApprovalStatusObj =
+          schemaRequestRepo.findOtherRequestorsSchemaRequestsGroupByStatus(
```

Comment:
why not to replace with count query?
Count query is much faster since it doesn't need to load all the field values

Reply:
So the actually query is a count, but it also pulled back the topicStatus, but because we are actually looking for a specific topicStatus we can remove that, and just get the count back and enter that correctly.

Making that change now.
Also did not accept the suggestions for the tests due to needing to sign the commits but I have included all of them in my branch with this change.